docs/issues.md

```diff
@@ -1,9 +1,9 @@
 # Filing An Issue
-The Openshift Aggregated Logging integration is a multicomponent technology stack whose individual 
-parts must all work together to gather and archive logs.  It is important to include as much information 
-as possible when issues arise in order to properly debug and diagnose.  Run 
-**ADD SCRIPT OR UTILITY HERE** which will gather the desired information.  Alternatively, you can gather 
-them manually. 
+The Openshift Aggregated Logging integration is a multicomponent technology stack whose individual
+parts must all work together to gather and archive logs.  It is important to include as much information
+as possible when issues arise in order to properly debug and diagnose.  Run
+[logging gathering script](../hack/dump/logging-dump.sh) which will gather the desired information.  Alternatively, you can gather
```

Comment:
I think the script should live in `/hack` - I don't see a reason to have a `/hack/dump` subdirectory.  This link should then just be `(/hack/logging-dump.sh)`